src/Agent.Plugins/Util/ExceptionsUtil.cs

```diff
@@ -0,0 +1,24 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Net.Sockets;
+using System.Text;
+using Microsoft.VisualStudio.Services.Agent;
+using Microsoft.VisualStudio.Services.Agent.Util;
+using Agent.Plugins.Log.TestResultParser.Contracts;
+
+namespace Agent.Plugins.Util
+{
+    public class ExceptionsUtil
+    {
+        public static void HandleSocketException(SocketException e, string url, ITraceLogger trace)
```

Comment:
You can define ExceptionsUtil once in Agent.Sdk project, and just import for others.

Reply:
Earlier I had issue with passing different types of tracers to handler, so several handlers were created as solution for different types of tracers. Currently I have 1 handler for all types of tracers. And it's solved.